app/view/component/MapController.js

```diff
@@ -14,6 +14,7 @@ Ext.define('momo.view.component.MapController', {
             var olMap = me.createOlMap();
             view.setMap(olMap);
         }
+        olMap.on('moveend',me.setRouting, me);
```

Comment:
Technically we should always do this if we bind on external things throwing events, e.g. ol.